astropy/io/fits/hdu/compressed/tests/test_compressed.py

```diff
@@ -204,6 +204,7 @@ def test_disable_image_compression(self):
         with fits.open(self.data("comp.fits")) as hdul:
             assert isinstance(hdul[1], fits.CompImageHDU)
 
+    @pytest.mark.slow
```

Comment:
Hmm, looking at the actual test, I think the only reason it is slow is because it *sleeps* for 1 second! Can we change that to 0.1 s?

Reply:
It works on my system but I'm not sure how portable this is, maybe the test would fail on slower archs/containers/VMs ? Anyway, `time.sleep(1)` is used in combinations with `pytest.mark.slow` in other similar tests, but admittedly a difference is that in this one case, `sleep` is actually responsible for most of the test's time, so let's try that.